core/util/src/main/java/org/arakhne/afc/progress/DefaultProgression.java

```diff
@@ -200,7 +200,7 @@ public int getMinimum() {
 	@Override
 	@SuppressWarnings("checkstyle:magicnumber")
 	public double getPercent() {
-		final double extent = this.max - this.min;
+		final double extent = this.max - (double) this.min;
```

Comment:
Not good because you force the usage of the double addition instead using the addition between integer before casting the resulting integer into a double.